Sources/CodeGen/LLVM/LLVMProgram.swift

```diff
@@ -55,7 +55,8 @@ public struct LLVMProgram {
   /// to `directory`, returning the URL of each written file.
   ///
   /// - Returns: The URL of each written product, one for each module in `self`.
-  public func write(_ type: SwiftyLLVM.CodeGenerationResultType, to directory: URL) throws -> [URL] {
+  public func write(_ type: SwiftyLLVM.CodeGenerationResultType, to directory: URL) throws -> [URL]
+  {
```

Comment:
```suggestion
  public func write(
    _ type: SwiftyLLVM.CodeGenerationResultType, to directory: URL
  ) throws -> [URL] {
```

It is unfortunate that swift-format doesn't apply this style (although it accepts it) but that's the one used consistently in the code base for parameter lists that exceed the character limit.